integration/auth/user_mgt_test.go

```diff
@@ -65,6 +66,14 @@ func testCreateUsers(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	// make sure that the user.TokensValidAvterTime is not in the future or stale.
+	if u.TokensValidAfterTime > time.Now().Unix() {
+		t.Errorf("timestamp cannot be in the future")
+	}
+	if u.TokensValidAfterTime < time.Now().Unix()-3600 {
```

Comment:
This is a new user right? What is the default TokensValidAfterTime for a new user?

Reply:
0